ui/src/app/images/types.ts

```diff
@@ -2,4 +2,6 @@ export type ImageValue = {
   arch: string;
   release: string;
   os: string;
+  subArch?: string;
+  title: string;
```

Comment:
I've updated the form images value type to include `subArch`, which is only relevant for non-Ubuntu images, and `title` so that we don't need to pass in Ubuntu releases or otherImages into the `ImagesTable` component